src/Api/Api.php

```diff
@@ -349,7 +353,10 @@ private function executeWithRetry(string $method, string $uri, array $params = [
             return false;
         }, self::RETRY_MAX_ATTEMPTS);
         $proxy = new RetryProxy($retryPolicy, $backOffPolicy);
-        return $proxy->call(function () use ($method, $uri, $params, $body) {
+        return $proxy->call(function () use ($method, $uri, $params, $body, $proxy) {
+            if ($proxy->getTryCount() > 1) {
```

Comment:
takhle neee... do toho RetryProxy pošli logger a ono to bude logovat automaticky
https://github.com/keboola/retry/blob/master/src/RetryProxy.php#L83

Reply:
Aha, super. Díky!